### ISSUE TITLE
Support updating of user details upon OIDC/SAML2/LDAP Login

### ISSUE THREAD

**Issue by danicc097** [BookStackApp/BookStack#3771]
### Describe the Bug

When user info is changed in the identity provider, changes are not reflected in the bookstack database. Only groups are synced at the moment AFAICT:

https://github.com/BookStackApp/BookStack/blob/a090720241ed3a0471a4dc9860b9f7cf8837f562/app/Auth/Access/Oidc/OidcService.php#L250-L265

### Steps to Reproduce

1. Register a new user in the IDP.
2. Login in Bookstack
3. Update the user in the IDP
4. Logout of Bookstack, clear cache and log and back in
5. Changes not reflected

### Expected Behaviour

I'd expect changes from the IDP to be reflected automatically

### Exact BookStack Version

22.09

### Hosting Environment

Ubuntu 20.04, docker image

**Comment by ssddanbrown** [comment]
Yeah, this is the case with all external auth options. I'm gonna re-categorise this as a feature request since it's not specifically a bug in existing supported functionality.

**Comment by Misaka-L** [comment]
@ssddanbrown Sorry to disturb, but this issue has been hanging on for 2 years and I want to ask if there is any progress on it?

**Comment by ssddanbrown** [comment]
@Misaka-L No, as above I see this as a feature request rather than an issue, so I'm not really aiming to address but more awaiting to validate as a need to justify the added desired complexity, for which there has been very little demand so far.

**Comment by ssddanbrown** [comment]
BookStack is now [managed on Codeberg](https://codeberg.org/bookstack/bookstack).

As part of this change, we're closing all existing GitHub issues and pull requests.
You'll instead find this issue on Codeberg here: 
https://codeberg.org/bookstack/bookstack/issues/3771
  
Any further discussion for this will now be done on Codeberg.
Therefore this GitHub thread will be locked.
